opendbc/car/hyundai/values.py

```diff
@@ -498,7 +498,7 @@ class CAR(Platforms):
   GENESIS_GV60_EV_1ST_GEN = HyundaiCanFDPlatformConfig(
     [
       HyundaiCarDocs("Genesis GV60 (Advanced Trim) 2023", "All", car_parts=CarParts.common([CarHarness.hyundai_a])),
-      HyundaiCarDocs("Genesis GV60 (Performance Trim) 2023", "All", car_parts=CarParts.common([CarHarness.hyundai_k])),
+      HyundaiCarDocs("Genesis GV60 (Performance Trim) 2022-23", "All", car_parts=CarParts.common([CarHarness.hyundai_k])),
     ],
     CarSpecs(mass=2205, wheelbase=2.9, steerRatio=12.6),  # steerRatio: https://www.motor1.com/reviews/586376/2023-genesis-gv60-first-drive/#:~:text=Relative%20to%20the%20related%20Ioniq,5%2FEV6%27s%2014.3%3A1.
```

Comment:
![image](https://github.com/user-attachments/assets/876a3965-c863-42e8-8f78-b42d036e9457)

This change will also affect the existing 2023 GV60, but I pulled its original test route, and it was at 16.7 and still rising toward the end, so this should be a good change for both.

```suggestion
    CarSpecs(mass=2205, wheelbase=2.9, steerRatio=17.6),
```